### DISCUSSION TITLE
Does main branch already support SmartBFT?

### DISCUSSION THREAD

**Discussion by bh4rtp2023** [hyperledger/fabric#4389] [Q&A] [answered]
Configtx.yaml has configuration for SmartBFT orderer. So the main branch supports SmartBFT now, doesn't it?

**Comment by tock-ibm** [comment, marked as answer]
Yes it does, but only partially, not all elements of a full BFT system are in place yet, and there needs to be more testing and documentation. There is work going on to complete that, see epic: https://github.com/hyperledger/fabric/issues/3771

**Comment by bh4rtp2023** [comment]
@tock-ibm hi, has 3.0.0-preview implemented [signature identity referencing](https://github.com/hyperledger/fabric-rfcs/blob/main/text/0010-bft-signatures.md#proposed-techniques) feature?

**Reply by bh4rtp2023** [reply]
I have tested with the same testsuite, the ledger size in /var/hyperledger/production/chains/chains/mychannel/:
fabric 2.5.4: 5.923MB
fabric 3.0.0-preview: 6.2499MB
